test/hotspot/jtreg/runtime/cds/appcds/SharedArchiveConsistency.java

```diff
@@ -34,42 +34,15 @@
  * @run main/othervm -Xbootclasspath/a:. -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI SharedArchiveConsistency on
  * @run main/othervm -Xbootclasspath/a:. -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI SharedArchiveConsistency auto
  */
+import jdk.test.lib.cds.CDSArchiveUtils;
 import jdk.test.lib.process.OutputAnalyzer;
 import jdk.test.lib.Utils;
```

Comment:
The above import is no longer needed.

Reply:
This one is new added util class, it is needed here. Do you think it is CDSTestUtils ?

Reply:
I was referring to `import jdk.test.lib.Utils;`. It was for `Utils.getRandomInstance();` and the code has been moved to the new class `CDSArchiveUtils`.

Reply:
I see, thanks. Will remove.